provider/qsd.py

```diff
@@ -297,11 +297,12 @@ def _fulfil_image_props(self, name, params):
             LOG_JOB.info("QSD skip to create image %s ", name)
         else:
             # Record the images are maintained by QSD.
-            LOG_JOB.info("QSD ready to create image %s", name)
             obj = qemu_storage.QemuImg(params, data_dir.get_data_dir(), name)
-            _, result = obj.create(params)
-            if result.exit_status != 0:
-                raise QsdError("Failed create image %s " % name)
+            if params.get("force_create_image", "yes") == "yes":
```

Comment:
force_create_image -> qsd_create_image, default value 'yes'

Reply:
line 243 remove_image -> qsd_remove_image, default value 'yes'

Reply:
updated it, thanks.